import/translation_maps/format_map.properties

```diff
@@ -28,6 +28,7 @@ Image               = Image
 InteractiveMultimedia = Software
 Journal             = Journal
 Kit                 = Kit
+LaserDisc           = Laser Disc
```

Comment:
You should also add `Laser Disc = "Laser Disc"` to `languages/en.ini` so this new format can be appropriately translated.